docs/integrations.md

```diff
@@ -12,24 +12,22 @@ see how to enable DocSearch on your website:
 
 - [Docusaurus][1] - [How to enable search][2]
 - [VuePress][3] - [Algolia Search][4]
-- [GitBook][5]
-- [pkgdown][6] - [DocSearch indexing][7]
-- [LaRecipe][8] - [Algolia Search][9]
-- [Orchid][10] - [Algolia Search][11]
+- [pkgdown][5] - [DocSearch indexing][6]
+- [LaRecipe][7] - [Algolia Search][8]
+- [Orchid][9] - [Algolia Search][10]
 
 If you're maintaining a similar tool and want us to add you to the list, get [in
```

Comment:
(Not exactly related to this PR.) Having the action verb in the link would make user engagement easier.

```suggestion
If you're maintaining a similar tool and want us to add you to the list, [get in
```